Dockerfile

```diff
@@ -101,8 +94,10 @@ RUN pip install --no-cache-dir --upgrade pip
 COPY --from=lambda-builder "${LAMBDA_TASK_ROOT}" "${LAMBDA_TASK_ROOT}"
 
 # install unixodbc and 'ODBC Driver 17 for SQL Server', needed for Azure Dedicated SQL Pools
+# install git needed for looker views collection
```

Comment:
Nit: It's used to collect more than looker views: https://github.com/monte-carlo-data/common/blob/9f1b1eb3d4caf19fa21524504597575d97b14562/montecarlodata_common/object_types.py#L8-L12

Reply:
The other types are retrieved using the API [here](https://github.com/monte-carlo-data/data-collector/blob/8931a42a4aee0e01a2b1a8c501caea5a15ca51e4/lambdas/clients/looker/looker_proxy_client.py#L112)